media/css/mozorg/history-slides.less

```diff
@@ -507,8 +507,10 @@
     }
 
     .logo {
+        .at2x('/media/img/logos/firefox/logo-quantum.png', 132px, 132px);
```

Comment:
It seems the Less mixin only outputs the background-size inside the high dpi media query, which means low-res screens aren't scaling the image. I think we just need to explicitly add background-size here as well.

![image](https://user-images.githubusercontent.com/205591/33788826-4cdfd208-dc29-11e7-9d06-7f9c442b024c.png)

Reply:
Ah, d'oh! Added (but can't concretely verify outside of dev tools, as I only have a high-dpi screen atm).

Reply:
That fixed it! And I vote we leave the Android one alone. We really need to get rid of this page anyway.